### PR TITLE
Allow to share the CA between servers

### PR DESCRIPTION
**Pull request by sileht**
Hi, 

This change allows to share the CA between servers.

Regards.

### CODE REVIEW

File: manifests/server.pp [hunk 1]
```diff
@@ -372,6 +377,7 @@
   $persist_tun = false,
   $server_poll_timeout = undef,
   $ping_timer_rem = false,
+  $shared_ca = undef
```
Comment (luxflux) [inline manifests/server.pp]: Please add a trailing comma so we have less changes next time.

**Comment by luxflux** [comment]
Thanks a lot for your pull request! I am very sorry for my late reply, I was too busy...

Could you write some tests for the client and the server? Could you also merge master as there have been some changes?

File: manifests/server.pp (hunk 1 above)
Comment (sileht) [inline manifests/server.pp]: Done

**Comment by sileht** [comment]
I have rebase my patch, and added some tests.

Cheers,

**Comment by luxflux** [comment]
Thanks for rebasing. Unfortunately there have been some more changes which produce merge conflicts. I am going to resolve them by myself via #130 as I don't want you to have to rebase again :smile:

**Pull request closed without merging**